guides/INSTALLING.md

```diff
@@ -310,7 +310,7 @@ All of the services in the [web](#web-journey), [publishing](#publishing-journey
 
 Use the following alternative commands:
 
-* [florence](https://github.com/ONSdigital/florence) - use: `$ make debug ENABLE_DATASET_IMPORT=true ENCRYPTION_DISABLED=true`
+* [florence](https://github.com/ONSdigital/florence) - use: `$ make debug ENCRYPTION_DISABLED=true ENABLE_HOMEPAGE_PUBLISHING=true`
```

Comment:
I will approve this PR as it is correct and good to have document, but the `ENABLE_HOMEPAGE_PUBLISHING` flag should likely always be true now in Florence as well.